pkg/spec/backup.go

```diff
@@ -66,26 +66,15 @@ func (bp *BackupPolicy) Validate() error {
 }
 
 type StorageSource struct {
+	// PV represents a Persistent Volume resource, operator will claim the
+	// required size before creating the etcd cluster for backup purpose.
+	// If the snapshot size is larger than the size specified, backup fails.
```

Comment:
backup fails -> operator would kill the cluster and report failure condition in status.

I guess the right direction is that we would change our ClusterStatus to be more k8s native and add a failure condition, reason for such.

Reply:
I just copied and moved the corrent comment from PVSource to StorageSource. Are you asking for some changes in it in this patch?

Reply:
yes.

Reply:
Yeah.. Please change it to
```
operator would kill the cluster and report failure condition in status.
```

Reply:
@hongchaodeng thanks, updated. It wasn't clear to me if you just wanted to update the comment (I wasn't aware of this behaviour) or you were talking about a future change and how it was related to this patch.